src/grpcClient.ts

```diff
@@ -35,32 +37,26 @@ export class TraderClient implements TraderClientInterface {
    * @param swapRequestSerialized
    */
   tradePropose(
```

Comment:
This method has been removed in tdex-protobuf v1 and should be delete here as well